newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -81,7 +81,13 @@ fun Routing.createSongRoutes() {
                         songId = songId,
                     )
                     streamData.cookies.forEach { cookie ->
-                        response.cookies.append(name = cookie.key, value = cookie.value, domain = "newm.io")
+                        response.cookies.append(
+                            name = cookie.key,
+                            value = cookie.value,
+                            domain = "newm.io",
```

Comment:
domain can be referenced from application.conf with a default value of "newm.io".